ait-extension/src/jsx/global.d.ts

```
@@ -0,0 +1,5 @@
+//@ts-ignore
+declare var JSON: {
```

Biome (1.9.4)

[error] Do not shadow the global "JSON" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)